build-tests.cmd

```diff
@@ -1,14 +1,21 @@
 @if "%_echo%" neq "on" echo off
-setlocal
+setlocal EnableDelayedExpansion
 
 set buildTests=build-tests.log
 echo Running build-tests.cmd %* > %buildTests%
 
+set options=/nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%buildTests%
```

Comment:
orthogonal but we should also use the binclash logger here like we do in build.cmd.